<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-2H7LN3B" ignored="4" name="Test" tests="3" failures="1" timestamp="13 Feb 2020 01:37:05 GMT" time="50.887" errors="0">
  <testcase name="TC_AT_LoginLogout_ValidTest" time="16.197" classname="com.actitime.loginlogout.TC_AT_LoginLogout_Valid"/>
  <testcase name="TC_AT_LoginLogout_InValid_Test" time="11.667" classname="com.actitime.loginlogout.TC_AT_LoginLogout_InValid">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.actitime.loginlogout.TC_AT_LoginLogout_InValid.TC_AT_LoginLogout_InValid_Test(TC_AT_LoginLogout_InValid.java:29)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- TC_AT_LoginLogout_InValid_Test -->
  <testcase name="TC_Customer_Create_Valid_Test" time="22.941" classname="com.actitime.customer.TC_Customer_Create_Valid"/>
  <testcase name="@BeforeSuite preCondition" time="0.0" classname="com.actitime.utilities.Config">
    <ignored/>
  </testcase> <!-- @BeforeSuite preCondition -->
  <testcase name="@BeforeSuite preCondition" time="0.0" classname="com.actitime.utilities.Config">
    <ignored/>
  </testcase> <!-- @BeforeSuite preCondition -->
  <testcase name="@AfterSuite afterSuiteMethod" time="0.0" classname="com.actitime.utilities.Config">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuiteMethod -->
  <testcase name="@AfterSuite afterSuiteMethod" time="0.0" classname="com.actitime.utilities.Config">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuiteMethod -->
</testsuite> <!-- Test -->
